brands/shop/variety_store.json

```diff
@@ -51,6 +51,17 @@
       "shop": "variety_store"
     }
   },
+  "shop/variety_store|Daiso Japan": {
+    "countryCodes": ["au"],
```

Comment:
From quick check at https://www.openstreetmap.org/search?query=Daiso+Japan#map=19/34.96736/136.61972 it seems to appear also in USA. Is it the same brand?

Reply:
I presume so, I've updated this.

For the record, I'm just adding what I know from local knowledge which is AU only, nothing stopping others following up later to expand the country codes with their local knowledge.